kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/service/impl/B2BPurchaseOrderSciquestServiceImpl.java

```diff
@@ -241,6 +267,12 @@ public String getCxml(PurchaseOrderDocument purchaseOrder, String requisitionIni
 
         return cxml.toString();
     }
+    
+    protected KualiDecimal calcTaxRate(KualiDecimal preTaxAmount, KualiDecimal taxAmount) {
+        if (taxAmount == null || preTaxAmount == null) { return KualiDecimal.ZERO; }
+        if (KualiDecimal.ZERO.equals(preTaxAmount)) { return KualiDecimal.ZERO; }

```

Comment:
Given our recent discussions of how we want to treat ported code from v3, let's just leave it alone. If there was a technical flaw that we could immediately see as a result of the code (e.g. high probability NPE non-handling etc.), then I'd say it's worth the time and effort. Here, however, we have a couple lines of poor readability, and we just don't have the time to rewrite the world at every turn.